### ISSUE TITLE
Set up Mistral locally: documentation

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#19]
## User story
1. As a dev team
2. I need to document how to run the Mistral LLM locally 
3. So that it remains as a backup option.

## Acceptance criteria
* Documentation of how to setup Mistral LLM

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
